{
  "Message": "The method or operation is not implemented.",
  "Data": {},
  "InnerException": null,
  "HelpLink": null,
  "Source": "Speckle.Sdk.Serialization.Tests",
  "HResult": -2147467263,
  "StackTrace": "   at Speckle.Sdk.Serialization.Tests.ExceptionSendCacheManager.SaveObject(String id, String json) in C:\\Users\\adam\\Git\\speckle-sharp-sdk\\tests\\Speckle.Sdk.Serialization.Tests\\Framework\\ExceptionSendCacheManager.cs:line 15\r\n   at Speckle.Sdk.Serialisation.V2.Receive.ObjectLoader.GetAndCache(String rootId, DeserializeProcessOptions options, CancellationToken cancellationToken) in C:\\Users\\adam\\Git\\speckle-sharp-sdk\\src\\Speckle.Sdk\\Serialisation\\V2\\Receive\\ObjectLoader.cs:line 70\r\n   at Speckle.Sdk.Serialisation.V2.Receive.DeserializeProcess.Deserialize(String rootId) in C:\\Users\\adam\\Git\\speckle-sharp-sdk\\src\\Speckle.Sdk\\Serialisation\\V2\\Receive\\DeserializeProcess.cs:line 42\r\n   at Speckle.Sdk.Serialization.Tests.ExceptionTests.<>c__DisplayClass4_0.<<Test_Exceptions_Receive_Cache>b__0>d.MoveNext() in C:\\Users\\adam\\Git\\speckle-sharp-sdk\\tests\\Speckle.Sdk.Serialization.Tests\\ExceptionTests.cs:line 105\r\n--- End of stack trace from previous location ---\r\n   at Xunit.Assert.RecordExceptionAsync(Func`1 testCode) in /_/src/xunit.assert/Asserts/Record.cs:line 114"
}
